Dockerfile

```diff
@@ -1,14 +1,7 @@
-FROM golang:1.11-alpine
+FROM alpine:3.9
 
-RUN apk --no-cache add ca-certificates make git && \
-  mkdir -p $GOPATH/src/github.com/txtdirect/txtdirect
+RUN apk --no-cache add ca-certificates
 
-WORKDIR $GOPATH/src/github.com/txtdirect/txtdirect
+ADD txtdirect /caddy
```

Comment:
Why add it as `/caddy`?

Reply:
It was `/caddy` in the previous version, so I thought maybe there is a reason for it :smile:

Reply:
Not that I know of. Might just have been an oversight from before :)